BREAKING.md

```diff
@@ -78,6 +78,7 @@ These packages were previously published under the `@fluidframework` scope:
 -   `@fluidframework/test-pairwise-generator`
 -   `@fluidframework/test-version-utils`
 -   `@fluidframework/test-loader-utils`
+-   `@fluidframework/test-end-to-end-tests`
```

Comment:
We already published 4.0.0, I don't think this is the right place to add this package. I think if we're being strict, the whole change should go to the next branch (and in main only the notice that we'll stop publishing it to npm in a future major release), but we can probably verify that none of our partners is using this particular package, so maybe we can be a bit aggressive with doing this in main. cc @ChumpChief for thoughts on that.

Reply:
@alexvy86 I think this is interesting because I can definitely see cases for why we may want to put this change in main or next. IMO it is okay to go into main since it won't actually "break" anything, since the only change is that we will no longer be publishing new versions of a published package. I'd love to hear other people's perspectives on this though